plugins/modules/proxmox_vm_info.py

```diff
@@ -34,11 +34,12 @@
   vmid:
     description:
       - Restrict results to a specific virtual machine by using its ID.
+      - If VM with the specified vmid does not exist in a cluster then resulting list will be empty.
     type: int
   name:
     description:
-      - Restrict results to a specific virtual machine by using its name.
-      - If multiple virtual machines have the same name then vmid must be used instead.
+      - Restrict results to a specific virtual machine(s) by using their name.
+      - If VM(s) with the specified name does not exist in a cluster then resulting list will be empty.
```

Comment:
I think this way the sentence sounds a bit better:
```suggestion
      - If VM(s) with the specified name do not exist in a cluster then the resulting list will be empty.
```